api/caching/tasks.py

```diff
@@ -158,7 +158,7 @@ def update_storage_usage_with_size(payload):
     provider = metadata.get('provider', 'osfstorage')
 
     target_file_id = metadata['path'].replace('/', '')
-    target_file_size = metadata.get('size', 0)
+    target_file_size = int(metadata.get('size', 0))
```

Comment:
drive-by: should we trust the 'sizeInt' that Water Butler already gives us instead?

Reply:
I suppose reading the whole ticket is beneficial! 👍